examples/tutorials/advance_model_config/02_device.ipynb

```diff
@@ -0,0 +1,528 @@
+{
```

Comment:
<p>You don;t need <code>before</code> pipeline here. Move everything to the  main pipleine.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/332/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>